packages/block-editor/src/hooks/test/supports.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Internal dependencies
+ */
+import {
+	hasBlockClassNameSupport,
+	hasVariationClassNameSupport,
+} from '../supports';
+
+describe( 'hasBlockClassNameSupport', () => {
```

Comment:
What problem do you anticipate with these tests? They document the behavior of the util. Should it be treated as private?

Reply:
No problem at all. I just thought we could skip them as the actual behavior is tested with the others. No strong opinion for sure.

Reply:
It actually took me a while to get `hasBlockClassNameSupport` and `hasVariationClassNameSupport` right, since they're a bit subtle, especially with regard to their default behavior (e.g. no `supports.className` field in `block.json` specified means block class name is `true`, whereas variation class name is `false`). I found that the tests helped me cover the various cases; at some point, I changed a default behavior, and the tests proved hugely helpful to get it right. So I'd like to keep them 😄